README.md

```diff
@@ -225,4 +220,25 @@ Each directory will be created with `cl-keytool.jar`, `cl-wallet.jar`, and a P12
 **In `code/data-l1`:**
 -   data-l1.jar     // The executable for the dL1 layer
 
+
+### `hydra remote_start`
+
+This method initiates the remote startup of your metagraph in one of the available networks: testnet, integrationnet, and mainnet.
```

Comment:
We should add here that the network is set in euclid.json under deploy->network.

Reply:
I agree